src/decode.h

```diff
@@ -250,7 +263,35 @@ static inline uint32_t sign_extend_b(const uint32_t x)
     return (int32_t) ((int8_t) x);
 }
 
+static inline bool queue_is_full(struct block_t *block)
```

Comment:
Since the operations are dedicated to basic blocks, the prefix `block_` should be enforced.